src/molecules/Modal/index.js

```diff
@@ -63,6 +73,11 @@ class Modal extends React.Component {
 
     document.body.style.overflow = this.state.initialBodyOverflow;
 
+    if (checkiOSDevice(navigator.userAgent)) {
```

Comment:
Cool. Had no idea what `navigator.userAgent` was, but just looked it up. Interesting!